src/main/java/org/logstash/beats/Message.java

```diff
@@ -0,0 +1,64 @@
+package org.logstash.beats;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Message implements Comparable<Message> {
+    private int sequence;
+    private String identityStream;
+    private Map data;
+    private Batch batch;
+
+    public Message(int sequence, Map map) {
+        setSequence(sequence);
```

Comment:
I think we can do without `setSequence` and `setData` and just set them directly in the constructor